configure.ml

```diff
@@ -924,7 +924,8 @@ let cflags_sse2 = ["-msse2"; "-mfpmath=sse"]
 let cflags, sse2_math =
   let _, slurp =
     (* Test SSE2_MATH support <https://stackoverflow.com/a/45667927> *)
-    tryrun "cc" (["-march=native"; "-dM"; "-E"]
+    let cc = try which "cc" with Not_found -> "gcc" in
+    tryrun cc (["-march=native"; "-dM"; "-E"]
                  @ cflags_sse2
                  @ [coqtop/"kernel/byterun/coq_interp.h"] (* any file *)) in
```

Comment:
In the stackoverflow link they use an empty stdin, can't we do that instead of doing some compilation in configure?
Or at least compile some dummy instead of something that looks like a real target.

Reply:
Yes, but it was simpler to implement with a dummy existing file, given option -E is:

``` -E Stop after the preprocessing stage; do not run the compiler proper. The output is in the form of preprocessed source code, which is sent to the standard output. 
```

Reply:
(otherwise another possible choice for the dummy C file would be `dev/header.c`)